src/nr_openai_observability/build_events.py

```diff
@@ -103,3 +110,45 @@ def build_error_events(request, error):
     )
 
     return {"messages": messages, "completion": completion}
+
+
+def build_embedding_event(response, request, response_headers):
+    embedding_id = str(uuid.uuid4())
+
+    embedding = {
+        "id": embedding_id,
+        "api_key_last_four_digits": f"sk-{response.api_key[-4:]}",
+        "timestamp": datetime.now(),
+        "response_time": response.response_ms,
+        "request.model": request.get("model"),
+        "response.model": response.model,
+        "usage.total_tokens": response.usage.total_tokens,
+        "usage.prompt_tokens": response.usage.prompt_tokens,
+        "api_type": response.api_type,
+        "vendor": "openAI",
```

Comment:
OpenAI + having it as Enum

Reply:
this is the same value used in the node SDK (uncapitalised), and also having an Enum for 1 value doesn't make the most sense, we can always use. an enum if we need more values there in the future